stories/atoms/components/fileInput/FileInput.stories.mdx

```diff
@@ -0,0 +1,479 @@
+import { ArgsTable, Meta, Story, Canvas } from '@storybook/addon-docs'
+import { FileInput } from 'ui/atoms/fileInput/FileInput'
+import { Typography } from 'ui/atoms/typography/Typography'
+import { ThemeProvider } from 'ui/atoms/theme/ThemeProvider'
+import { ThemeSwitcher } from 'ui/atoms/theme/ThemeSwitcher'
+import { useState } from 'react'
+
+<Meta
+  title="Atoms/FileInput"
+  component={FileInput}
+  parameters={{
+    actions: { argTypesRegex: '^on.*' },
+    docs: {
+      source: {
+        type: 'code',
+        format: true,
+        language: 'jsx'
+      }
+    }
+  }}
+  decorators={[
+    Story => (
+      <ThemeProvider>
+        <div className="component-story-main">
+          <ThemeSwitcher />
+          <div
+            style={{
+              padding: '10px 0',
+              display: 'flex',
+              flexDirection: 'column',
+              alignItems: 'center'
+            }}
+          >
+            <Story />
+          </div>
+        </div>
+      </ThemeProvider>
+    )
+  ]}
+/>
+
+# FileInput
+
+> Allows the user to drag and drop or browse one or several files.
+
+[![stability-unstable](https://img.shields.io/badge/stability-unstable-yellow.svg)](https://github.com/emersion/stability-badges#unstable)
+
+## Overview
+
+### Description
+
+`FileInput` allows the user to drag and drop or browse one or several files before upload these files for example.
+
+<Canvas>
```

Comment:
If you don't mind, I'd like to remove here the `<Canvas>` tag because the associated code snippet outputs a fonction using `args` which has no real sense for the user.

Reply:
We talked about keep the `<Canvas>` tag for the user can access the `Show code` button.

Reply:
@JoeyDjr 
You are right but, in my opinion we should disable the `<Canvas>`element only for overview story as the code snippet outputs the spread  `args` that are confusing for the user: 
![image](https://user-images.githubusercontent.com/47554752/178465119-c45ed7dd-0dcc-41ba-889f-54a0a565b567.png)

Reply:
ok you're right